website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -37,6 +39,10 @@
                     <td>
                         Now that you've shared your ${reviewable.provider.preprint_word}, take advantage of more OSF features:
                         <ul>
+                            % if has_psyarxiv_chronos_text:
+                                <li>Submit your preprint to an APA-published journal. <a href=“https://help.osf.io/hc/en-us/articles/360044820913-Submit-to-APA-Journals-via-PsyArXiv”>Learn more</a></li>
```

Comment:
Are those smart quotes on the href?

Reply:
Yes, amazing catch, I have no idea how or they are I there I must of copy and pasted from something with smart quotes at some point. But anyway that's an excellent catch thank you I could have fished around all week and never caught that.